cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -61,12 +61,4 @@ spec:
         - containerPort: 53
           name: dns-tcp
           protocol: TCP
-        livenessProbe:
```

Comment:
We only change manfest file here, why requires version change here? If it does, we already messed up with https://github.com/GoogleCloudPlatform/kubernetes/commit/988aa6fdf67110cdfc012bf79a15d39c337f58bc then.